torch/csrc/api/include/torch/nn/functional.h

```diff
@@ -7,3 +7,4 @@
 #include <torch/nn/functional/normalization.h>
 #include <torch/nn/functional/pooling.h>
 #include <torch/nn/functional/vision.h>
+#include <torch/nn/functional/batchnorm.h>
```

Comment:
It would be awesome to put it above `#include <torch/nn/functional/distance.h>`, to sort them alphabetically :D

Reply:
OK. I followed your instruction.